lab5.md

```diff
@@ -34,6 +34,15 @@ $ docker run -p 4040:4040 -d --name=ngrok --net=func_functions \
   alexellis2/ngrok-admin http gateway:8080
 ```
 
+> Note: If you're running on Kubernetes, use this command instead:
+>
+> ```
+> kubectl -n openfaas_fn run \
+> --image=alexellis2/ngrok-admin \
+> --port=4040 \
+> ngrok
+> ```
+
```

Comment:
Only doing above doesn't work. We probably need to do something like the following:

```
kubectl -n openfaas run \
--image=alexellis2/ngrok-admin \
--port=4040 \
ngrok -- http gateway:8080

kubectl -n openfaas expose deployment ngrok \
--type=NodePort \
--name=ngrok

POD=$(kubectl get pods -n openfaas -l run=ngrok -o jsonpath="{.items[0].metadata['name']}")
kubectl -n openfaas port-forward $POD 4040:4040
```